src/main/java/com/cleanroommc/groovyscript/compat/mods/draconicevolution/DraconicEvolution.java

```diff
@@ -10,7 +10,5 @@ public class DraconicEvolution extends ModPropertyContainer {
 
     public DraconicEvolution() {
         this.energyCore = GroovyScriptConfig.compat.draconicEvolutionEnergyCore ? new EnergyCore() : null;
```

Comment:
a bit late here, but would adding an `isEnabled` checking `GroovyScriptConfig.compat.draconicEvolutionEnergyCore` be able to replace this?